### PR TITLE
Semester and Topic page

### PR DESCRIPTION
**Pull request by HrishB16**
Please review and merge it

### CODE REVIEW

**Comment by codadept** [comment]
Take a pull from `dev` branch and push the merge commit

**Comment by resyfer** [comment]
Semester Page:

Laptop:
image

Mobile:
image

image

Same issue for Topics page.

Fix it.

Also change (yet again) every absolute unit in the css you wrote to a relative unit.

**Review by resyfer — CHANGES_REQUESTED** [review]
Topics:

image

Scrollbar at right within the topics box. Also remove horizontal. Fix it. Do NOT remove the scrollbar using display property. Instead increase the height (dynamically, to have as many items as needed in it), and remove horizontal scrollbar by fixing the main issue in it.

Same issues with Semester

File: src/Components/Semester/Semester.module.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .container {
   height: 100vh;
-  width: 100vw;
+  width: 98vw;
```
Comment (resyfer) [inline src/Components/Semester/Semester.module.scss]: Changing 100vw to 98vw for removing horizontal scrollbar is absurd. Use 100% for width.

Please google stuff like this before using maths to accomplish it, as that maths will most likely only work on your PC.

File: src/Components/Semester/Semester.module.scss [hunk 2]
```diff
@@ -160,7 +160,8 @@
 
 @media only screen and (max-width: 1200px) {
   .container {
-    width: 100%;
+    width: 100vw;
```
Comment (resyfer) [inline src/Components/Semester/Semester.module.scss]: Same issue here

File: src/Components/Topics/Topics.module.scss [hunk 3]
```diff
@@ -1,5 +1,6 @@
 .container {
   height: 100vh;
+  width: 98vw;
```
Comment (resyfer) [inline src/Components/Topics/Topics.module.scss]: Again

File: src/Components/Topics/Topics.module.scss [hunk 4]
```diff
@@ -149,11 +150,11 @@
 
 @media only screen and (max-width: 1200px) {
   .container {
-    width: 100%;
+    width: 100vw;
```
Comment (resyfer) [inline src/Components/Topics/Topics.module.scss]: And again

**Review by resyfer — CHANGES_REQUESTED** [review]

**Pull request merged**